### ISSUE TITLE
Docs: demo buttons should be really hidden when not in use

### ISSUE THREAD

**Issue by marcysutton** [angular/material#240]
As a keyboard or screen reader user, I should not be able to tab onto buttons that are visually hidden. For example: as a sighted mouse user, when I click "Show Demo Source", I am presented with tabs containing component markup, scripts, and CSS. These buttons and content panels should not be in the tab order or read aloud when "Show Demo Source" is inactive.

Screenshot shows keyboard focused on `index.html` button, should not be allowed:
![Focus on hidden button](https://cloud.githubusercontent.com/assets/1045233/4142285/f9948ac6-33b6-11e4-922c-fa52f54a6d85.png)

Screenshot shows expanded demo source, including buttons in the tab order:
![Expanded demo source](https://cloud.githubusercontent.com/assets/1045233/4142341/c44eaf6c-33b7-11e4-9493-1bc7199d74c1.png)
